backend/capellacollab/core/metadata.py

```diff
@@ -5,20 +5,34 @@
 import pydantic
 
 import capellacollab
+from capellacollab.config import config
 
 
 class Metadata(pydantic.BaseModel):
     model_config = pydantic.ConfigDict(from_attributes=True)
 
     version: str
+    privacy_policy_url: str | None
+    imprint_url: str | None
+    provider: str | None
+    authentication_provider: str | None
+    environment: str | None
```

Comment:
There is a mismatch between the type in the config schema and this. I would either adjust the config schema to also allow an empty property (I think you need to add "null" as additional type to allow this) or remove the `| None` here.

(This also hold for the `cfg: dict[str, str | None]`, i.e., either just use `str` as value type or adjust the schema)

Reply:
The configuration schema doesn't allow `None`, but it allows that the key itself is missing. It is not required in the YAML schema. Therefore, it `None` is used if the configuration option is missing (to provide compatibility downwards).